### PR TITLE
Hallowed Ground 155646.

### PR DESCRIPTION
**Pull request by Ulduar2**
Honestly, where did I get this:
https://github.com/Kittnz/FirestormWoD/blob/master/src/server/scripts/Draenor/Auchindoun/boss_kaathar.cpp#L805
**Changes proposed:**

-  Attempt summon creature. This creature should cast fissure on self. Areatrigger 2647 (SpellID 165065) already have data in ADB 7.3.5.
-  
-  

**Issues addressed:** Closes #  (insert issue tracker number)
No

**Tests performed:** (Does it build, tested in-game, etc.)
Sorry, but no.

**Known issues and TODO list:** (add/remove lines as needed)

- [ ] 
- [ ] 

http://www.wowhead.com/spell=155646/hallowed-ground
``` SQL
Targets Mask = 0x00000040 (TARGET_FLAG_DEST_LOCATION)
Spell Level = 0, base 0, max 0

Category = 0
DispelType = 0 (DISPEL_NONE)
Mechanic = 0 (MECHANIC_NONE)
SpellRange: (Id 13) "Anywhere":
    MinRangeNegative = 0, MinRangePositive = 0
    MaxRangeNegative = 50000, MaxRangePositive = 50000

Speed 3,00
Cast time (Id 1) = 0,00

Interrupt Flags: 0x00000000, AuraIF 0x00000000, ChannelIF 0x00000000
Chance = 0, charges - 0
=================================================
Effect 0: Id 32 (SPELL_EFFECT_TRIGGER_MISSILE) DIFFICULTY_NONE
BasePoints = 0
Targets (138, 91) (TARGET_UNK_138, TARGET_DEST_DEST_RADIUS)
Max Radius (Id 74) 0,10
Trigger spell (169753) Not found, Chance = 0
```

### CODE REVIEW

**Comment by Traesh** [comment]
Hello, thanks for the PR :)

It seems that this spell is used by "Vigilant Kaathar" only in Auchindoun, could you please switch your spell to this boss script cpp instead of spell_generic ?

**Comment by Ulduar2** [comment]
Hello, @Traesh 
Sorry, but Auchindoun not founded files in repository.
Unfortunately, I simply have nowhere to place.

File: sql/ashamane/world/2018_04_24_18_HallowedGround.sql [hunk 1]
```diff
@@ -0,0 +1,3 @@
+DELETE FROM spell_script_names WHERE scriptname IN ("spell_hun_exhilaration");
```
Comment (HannibalRoG) [inline sql/ashamane/world/2018_04_24_18_HallowedGround.sql]: Why?

Comment (Ulduar2) [reply sql/ashamane/world/2018_04_24_18_HallowedGround.sql]: Sorry, repaired.

**Comment by Traesh** [comment]
I will probably take aunchindoun from FirestormWod repository, and because this spell is only used in auchindoun i think it's better to wait for it than merge spell by spell

**Comment by Ulduar2** [comment]
Okay

**Pull request closed without merging**